dot/rpc/modules/system_test.go

```diff
@@ -230,10 +233,73 @@ func TestSystemModule_Version(t *testing.T) {
 }
 
 func TestSystemModule_Properties(t *testing.T) {
-	sys := NewSystemModule(nil, newMockSystemAPI())
+	sys := NewSystemModule(nil, newMockSystemAPI(), nil, nil, nil)
+
 	expected := map[string]interface{}(nil)
+
 	res := new(interface{})
 	err := sys.Properties(nil, nil, res)
 	require.NoError(t, err)
 	require.Equal(t, expected, *res)
 }
+
+func TestSystemModule_AccountNextIndex(t *testing.T) {
```

Comment:
is it possible to add a test for the non-pending case?

Reply:
Yes, I've added tests.